subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/recomp/RecompilationSpec.java

```diff
@@ -23,14 +23,10 @@
 import java.util.LinkedHashSet;
 import java.util.Set;
 
-import static org.gradle.api.internal.tasks.compile.incremental.recomp.WellKnownSourceFileClassNameConverter.isModuleInfo;
-import static org.gradle.api.internal.tasks.compile.incremental.recomp.WellKnownSourceFileClassNameConverter.isPackageInfo;
-
 public class RecompilationSpec {
     private final Set<String> classesToCompile = new LinkedHashSet<>();
     private final Collection<String> classesToProcess = new LinkedHashSet<>();
     private final Collection<GeneratedResource> resourcesToGenerate = new LinkedHashSet<>();
-    private final Set<String> relativeSourcePathsToCompile = new LinkedHashSet<>();
```

Comment:
This is no longer needed, we now use the SourceToClassName converter consistently for everything.